### PR TITLE
Add support for `Date` objects to `any`

### PR DESCRIPTION
**Pull request by lejeunerenard**
Encodes `Date` objects by encoding them as their ISO 8601 format string.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -573,6 +573,19 @@ const anyObject = {
...
+const anyDate = {
+  preencode (state, o) {
+    utf8.preencode(state, o.toISOString())
+  },
+  encode (state, o) {
+    utf8.encode(state, o.toISOString())
+  },
+  decode (state) {
+    const isoString = utf8.decode(state)
+    return new Date(isoString)
```
Comment (mafintosh) [inline index.js:585]: lets encode as uint, seems much smaller

Comment (mafintosh) [reply index.js:585]: ie o.getTime() and new Date(ms)

Comment (mafintosh) [reply index.js:585]: did it for you in 2.16.0 \o/

Comment (lejeunerenard) [reply index.js:585]: Agreed, `unit` for ms is much smaller and better.

**Pull request closed without merging**